content/blog/elm-review-v2/index.md

```diff
@@ -0,0 +1,269 @@
+- [`jfmengels/review-common`](https://package.elm-lang.org/packages/jfmengels/review-common/latest/)
+- [`jfmengels/review-debug`](https://package.elm-lang.org/packages/jfmengels/review-debug/latest/)
+
+I am also working on [a package to improve the quality of the documentation](https://github.com/jfmengels/review-documentation), which should help out package authors especially. And there are also some rules that I wrote, but am still unsure as to whether I want to maintain them personally, that you can copy over from [`jfmengels/review-simplification`](https://github.com/jfmengels/review-simplification).
+
+## How does this compare to Elm Analyse?
+
+##### Philosophy-wise
+
+`Elm Analyse` is at the moment the de facto static code analysis tool for Elm, but it has [a different philosophy](https://stil4m.github.io/elm-analyse/#/contributing) from `elm-review`'s. `Elm Analyse` wants to improve the quality of the code by enabling rules that work for "everyone".
+
+`elm-review` on the other hand aims to create guarantees tailored to your team and project, while enabling ways to improve the quality of the code too in a shareable manner.
+
+If you think you have a rule that could be use to everyone, you can share it by publishing it in the Elm package registry.
+
+##### Functionality-wise
+
+Most checks (their naming for a rule) you can find in `Elm Analyse` are available in the packages I published or GitHub repos I have written. The remaining ones are ones that are outdated or that I disagree with, but these are reasonably easy to write using this package's API. (Exception for the checks for unused patterns and unused arguments which I plan to add). And anyone can publishing the missing ones if they care to.
+
+From the tests I have run, I found `elm-review` to be faster. I am guessing that that is mostly because rules are built in a way that avoid duplicate work.
+
+Here are the things that `Elm Analyse` has and `elm-review` hasn't:
+
+- There is no web interface, but there is a similar CLI watch mode.
+- `elm-review` doesn't show the graph of modules, but I don't really see the value that brings.
+- `elm-review` doesn't show when dependencies can be updated. I see the value, but I don't think it is a good fit for the tool.
+- There is no editor support for `elm-review`, but let me know if you want to help out with that!
+
+As for the rest, I hope to have shown in this post and in the [original announcement](/announcing-elm-review/) what this tool can do that `Elm Analyse` can not.
+
+## Get started!
+
+If you already use `elm-review` in your projects, you can follow this [migration guide](https://github.com/jfmengels/elm-review/blob/master/documentation/Migration%20v1%20to%20v2.md). Otherwise, follow these instructions!
+
+```bash
+cd your-project/
+
+# Using npm
+npm install --save-dev elm-review
+# Using Yarn
+yarn add --dev elm-review
+
+# Create a configuration
+npx elm-review init
```

Comment:
What's `npx`?

Reply:
It's a tool that comes with `npm` and allows you to run binaries from the `node_modules` without adding them to your PATH